zkevm-circuits/src/evm_circuit.rs

```diff
@@ -1 +1,930 @@
+                    (expr.clone(), meta.query_fixed(*column, Rotation::cur()))
+                })
+                .collect::<Vec<_>>()
+            });
+        }
+
+        // Configure fixed lookups
+        for fixed_lookup in fixed_lookups.iter() {
+            meta.lookup(|meta| {
+                fixed_lookup
+                    .iter()
+                    .zip(fixed_table.iter())
+                    .map(|(expr, column)| {
+                        (
+                            expr.clone(),
+                            meta.query_fixed(*column, Rotation::cur()),
+                        )
+                    })
+                    .collect::<Vec<_>>()
+            });
+        }
+
+        // Configure rw lookups
+        for rw_lookup in rw_lookups.iter() {
+            meta.lookup(|meta| {
+                rw_lookup
+                    .iter()
+                    .zip(rw_table.iter())
+                    .map(|(expr, column)| {
+                        (
+                            expr.clone(),
+                            meta.query_advice(*column, Rotation::cur()),
+                        )
+                    })
+                    .collect::<Vec<_>>()
+            });
+        }
+    }
+
+    fn load_fixed_tables(
+        &self,
+        layouter: &mut impl Layouter<F>,
+    ) -> Result<(), Error> {
+        layouter.assign_region(
+            || "fixed table",
+            |mut region| {
+                let mut offset = 0;
+
+                // Noop
+                for (idx, column) in self.fixed_table.iter().enumerate() {
+                    region.assign_fixed(
+                        || format!("Noop: {}", idx),
+                        *column,
+                        offset,
+                        || Ok(F::zero()),
+                    )?;
+                }
+                offset += 1;
+
+                // Range256
+                for idx in 0..256 {
+                    region.assign_fixed(
+                        || "Range256: tag",
+                        self.fixed_table[0],
+                        offset,
+                        || Ok(F::from_u64(FixedLookup::Range256 as u64)),
+                    )?;
+                    region.assign_fixed(
+                        || "Range256: value",
+                        self.fixed_table[1],
+                        offset,
+                        || Ok(F::from_u64(idx as u64)),
+                    )?;
+                    for (idx, column) in
+                        self.fixed_table[2..].iter().enumerate()
+                    {
+                        region.assign_fixed(
+                            || format!("Range256: padding {}", idx),
+                            *column,
+                            offset,
+                            || Ok(F::zero()),
+                        )?;
+                    }
+                    offset += 1;
+                }
+
+                // Range32
+                for idx in 0..31 {
```

Comment:
Thanks to @therealyingtong to point out the range error here.

```suggestion
                for idx in 0..32 {
```